test/unit/text.js

```diff
@@ -394,6 +394,13 @@
     assert.deepEqual(obj.styles, [], 'empty style object has been removed');
   });
 
+  QUnit.test('text toObject can handle style objects with only a textBackgroundColor property', function(assert) {
+    var text = new fabric.Text('xxx');
+    text.styles = { 0: { 0: { textBackgroundColor: 'blue' } } };
+    var obj = text.toObject();
+    assert.deepEqual(obj.styles, [{ start: 0, end: 1, style: { textBackgroundColor: 'blue' }}], 'empty style object has been removed');
```

Comment:
Thanks for the PR!

Just a couple things. This message should be updated though to something that fits the test. Also, PRs now require adding an entry to the changelog (https://github.com/fabricjs/fabric.js/blob/master/CHANGELOG.md). Everything else looks good to me!

Reply:
whoops! totally missed that I'll get a fix up for that